src/com/egs/Hovhannes/Task1/SymbolRepeat/SymbolRepeat.java

```diff
@@ -0,0 +1,143 @@
+package EGS.SymbolRepeat;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class SymbolRepeat {
+
+    class Line {
+        private String symbol = "";
+        private int lineNumber = 0;
+        private int count = 0;
+
+        @Override
+        public String toString() {
+            return "Line{" +
+                    "symbol='" + symbol + '\'' +
+                    ", lineNumber=" + lineNumber +
+                    ", count=" + count +
+                    '}';
+        }
+
+        public Line() {
+        }
+
+        public Line(String symbol) {
+            this(symbol, 0, 0);
+        }
+
+        public Line(String symbol, int lineNumber, int count) {
+            this.symbol = symbol;
+            this.lineNumber = lineNumber;
+            this.count = count;
+        }
+
+        public String getSymbol() {
+            return symbol;
+        }
+
+        public void setSymbol(String symbol) {
+            this.symbol = symbol;
+        }
+
+        public int getLineNumber() {
+            return lineNumber;
+        }
+
+        public void setLineNumber(int lineNumber) {
+            this.lineNumber = lineNumber;
+        }
+
+        public int getCount() {
+            return count;
+        }
+
+        public void incrementCount(int count) {
+            this.count = count;
+        }
+    }
+
+    private String toFind = "";
+    static String inFileName = "";
+    static String outFileName = "";
+    private File file = null;
+    private List<SymbolRepeat.Line> lines = new ArrayList<>();
+
+    public void process() {
+
+        file = new File(inFileName);
+
+        if (file.exists()) {
+            try (FileReader r = new FileReader(file);) {
+
+                BufferedReader br = new BufferedReader(r);
+
+                String tmpLine = "";
+                String tmpWords[];
+                int wordCount = 0;
+                int lineNumber = 1;
+
+                while (br.ready()) {
+                    tmpLine = br.readLine();
+
+                    tmpWords = tmpLine.split(" ");
+
+                    for (String word : tmpWords) {
+                        if (word.equalsIgnoreCase(toFind))
+                            wordCount++;
+                    }
+
+                    lines.add(new Line(toFind, lineNumber, wordCount));
+                    lineNumber++;
+                    wordCount = 0;
+                }
+                r.close();
+                br.close();
+
+            } catch (IOException ex) {
+                System.out.println(ex.getMessage());
+                System.out.println(ex.getStackTrace());
```

Comment:
stacktrace will print the log anyway
getMessage is not necessary, if not mistaken stacktrace contains message as well